Makefile

```diff
@@ -6,26 +6,54 @@
 # This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not
 # distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 #
-# date: 2015-10-08
+# date: 2015-10-11
 #
 
 #-----------------------------------------------------------------------------------------------------------------------
+# make control/macros
+#-----------------------------------------------------------------------------------------------------------------------
+# defaults
+mk_do_inc := 1
```

Comment:
I see there a big difference between real variables, containing build related information like
CFLAGS, OUTPUT, PROJECT, ... , and such variables to control the makefile itself. This was
the reason to prefix them with "mk_". Look to the variables used in  "PARSE_SUBDIRECTORY".
There you have dir, sp and d.
It is not easy to know for other people, which syntax you like for this and for another thing. Even
in your own makefile you aren't consistent!

Please provide clear rules for this in your project.